resources/views/components/debt-table.blade.php

```diff
@@ -10,55 +10,57 @@ class="mb-6 px-4 py-3 bg-grey-blue hover:bg-halloween-orange no-underline rounde
     @endif
 </div>
 
-<div class="rounded-lg shadow">
-    <ul class="bg-grey-blue-light flex p-4 rounded-t-lg border-grey border-b-2">
-        <li class="w-2/7 text-grey-darkest font-bold uppercase text-xs leading-none tracking-wide">Project Name</li>
+<table class="table-auto rounded-lg shadow max-w-full overflow-x-scroll">
```

Comment:
See meta PR comment for why I'm suggesting this

```suggestion
<div class="overflow-x-auto max-w-full">
<table class="table-auto rounded-lg shadow max-w-full">
```